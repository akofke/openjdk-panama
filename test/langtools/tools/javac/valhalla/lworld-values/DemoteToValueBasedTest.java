/*
 * @test /nodynamiccopyright/
 * @summary Test that value classes get demoted to value based classes in -source 10 compiles.
 * @compile Point.java
 * @compile/fail/ref=DemoteToValueBasedTest.out -XDrawDiagnostics -XDdev DemoteToValueBasedTest.java
 * @compile/fail/ref=DemoteToValueBasedTest10.out --should-stop=at=FLOW -Werror -XDrawDiagnostics -source 10 -XDdev DemoteToValueBasedTest.java
 */

public class DemoteToValueBasedTest {
    Point p = null;
}
