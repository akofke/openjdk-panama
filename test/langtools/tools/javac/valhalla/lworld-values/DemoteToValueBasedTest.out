DemoteToValueBasedTest.java:11:15: compiler.err.prob.found.req: (compiler.misc.inconvertible.types: compiler.misc.type.null, Point)
1 error
